restructuredtext_lint/cli.py

```diff
@@ -12,7 +12,7 @@
     VERSION = version_file.read().strip()
 
 
-def _main(filepaths, format='text', stream=sys.stdout, encoding=None):
+def _main(filepaths, format='text', stream=sys.stdout, encoding=None, level=0, fail=0):
```

Comment:
I think there was some talking past each other. In https://github.com/twolfson/restructuredtext-lint/pull/37#issuecomment-281956861 I talked about dropping `--fail` (although I see that I never mentioned it explicitly). I would like to only have `--level` as I don't feel `--fail` is a common use case. If someone wants to do something custom, then they will likely use `--level` and ignore our exit code